packages/components/stories/TabBar.js

```diff
@@ -28,7 +28,12 @@ const props = {
 	selected: '2',
 };
 
-storiesOf('TabBar', module)
+const stories = storiesOf('TabBar', module);
+if (!stories.addWithInfo) {
+	stories.addWithInfo = stories.add;
+}
```

Comment:
addWithInfo is a plugin which add stuff in the rendering of the component.
I want to remove it (as mentionned in the PR description) so the output is as most as possible the component output.
so if this plugin is not activated wchich is the case in storyshots now we don't have addWithInfo